be/src/common/status.h

```diff
@@ -466,7 +466,7 @@ class [[nodiscard]] Status {
     ERROR_CTOR(MinimumReservationUnavailable, MINIMUM_RESERVATION_UNAVAILABLE)
     ERROR_CTOR(Corruption, CORRUPTION)
     ERROR_CTOR(IOError, IO_ERROR)
-    ERROR_CTOR(NotFound, NOT_FOUND)
+    ERROR_CTOR_NOSTACK(NotFound, NOT_FOUND)
```

Comment:
better use some thing like  `return Status::Error<ErrorCode::NOT_FOUND, false>(
                    "not found txn info, tablet_id={}, transaction_id={}", tablet_id,
                    transaction_id);` , maybe some error need to print stack